tests/testPsfFitter.py

```diff
@@ -123,7 +123,9 @@ def testFixedModel(self):
         ellipses3 = model.makeEllipseVector()
         for i in range(len(ellipses2)):
             self.assertFloatsAlmostEqual(ellipses1[i].getParameterVector(), ellipses2[i].getParameterVector())
-            ellipses3[i] = msf.getComponents()[i].getEllipse()  # need to convert ellipse parametrization
+            # need to convert ellipse parametrization
+            ellipses3[i].setCore(msf.getComponents()[i].getEllipse().getCore())
+            ellipses3[i].setCenter(msf.getComponents()[i].getEllipse().getCenter())
```

Comment:
Why doesn't the old code work? Is it because python doesn't have C++-like assignment? Again, I don't see any changes in the wrapping that are related.

Same question for the two analogous blocks later in the file.